x-pack/platform/plugins/shared/cases/server/features/v1.ts

```diff
@@ -71,10 +72,10 @@ export const getV1 = (): KibanaFeatureConfig => {
         },
         ui: capabilities.all,
         replacedBy: {
-          default: [{ feature: FEATURE_ID_V2, privileges: ['all'] }],
+          default: [{ feature: FEATURE_ID_V3, privileges: ['all'] }],
           minimal: [
             {
-              feature: FEATURE_ID_V2,
+              feature: FEATURE_ID_V3,
               privileges: ['minimal_all', 'create_comment', 'case_reopen'],
```

Comment:
Question: Should `case_assign` be added here since it is now split out in V3?

Reply:
Yes, added. At first I thought it would be just to v2, but integration tests indicate it should be in both.